### ISSUE TITLE
Open source the ecosystem map

### ISSUE THREAD

**Issue by cassshih** [EthereumCommunityFund/ethereum-ecosystem-map#1]
@awasunyin we can use this map/taxonomy as a starting point and open source it later, inviting the community to add what they are building to this list.

**Comment by awasunyin** [comment]
Yes! I want to do a couple more iterations, and write up the initial list so people have examples before they open issues for this
